### PR TITLE
Task8

### PR DESCRIPTION
**Pull request by niravpansuriya53**

### CODE REVIEW

File: task8.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+'''
+
+numbers=[2, 5, 6, 1, 3, 8, 9, 10]
+
+#shorted list
+for i in range(0,len(numbers)):
+    for j in range(i+1,len(numbers)):
+        if numbers[i] > numbers[j]:
+            numbers[i], numbers[j] = numbers[j], numbers[i]
+print("sorted list is :- ",numbers)
```
Comment (himanshu-suthar-tagline) [inline task8.py]: Please tell me the name of algorithm you have used for sorting the list.

Comment (niravpansuriya53) [reply task8.py]: swapping without third variable

Comment (niravpansuriya53) [reply task8.py]: a=10
b=20
swapping  two number without third variable

a = a+b
b = a-b
a = a-b

than a=20 b=10

Comment (himanshu-suthar-tagline) [reply task8.py]: I didn't ask for the logic. Would you please tell me which `sorting algorithm` is this? I want name of sorting algorithm.

Comment (niravpansuriya53) [reply task8.py:16]: Bubble Sort

Comment (niravpansuriya53) [reply task8.py]: Selection Sort

Comment (himanshu-suthar-tagline) [reply task8.py]: It is bubble sort. Now implement your code based on selection sort.

**Pull request merged**